<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidgetClass</class>
 <widget class="QMainWindow" name="AdminWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Date of event</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="event_date_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="event_link_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Description of event</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="event_description_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="event_title">
       <property name="text">
        <string>Title of event</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="event_title_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="list_widget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>1</y>
      <width>361</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_event_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_event_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_event_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of people</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="event_number_of_people_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
